packages/glyphs-block/src/GlyphsBlock.tsx

```diff
@@ -40,7 +46,18 @@ export const GlyphsBlock = ({ appBridge }: BlockProps): ReactElement => {
                         outlineWidth: borderWidth,
```

Comment:
you could take all the border infos into one css: `outline: `${borderWidth} ${borderStyle} ${toRgbaString(borderColor)}`,`

Reply:
Changed 👍🏻